docs/quick_start.mdx

```diff
@@ -221,22 +221,30 @@ npm install -g electron firebase-tools dotenv-cli
 
 ## Run the Task
 
-Running the task in development mode causes it to hot-reload whenever changes are made to the app. This is how you'll run the project while building your task.
-
-```shell title="Run the task in dev mode"
+```shell title="Run the task in development mode"
 npm run dev
 ```
 
-This script launches an Electron window with the task and inspector open.
+Running the task in development mode causes allows for hot-reloading whenever changes are made to the app. This is how you'll run the project while creating your task!
+
+:::tip
+The dev script runs Honeycomb on Electron without any environment variables. Check out the [NPM Scripts](npm_scripts#dev) page for more information on the available development environments.
+:::
+
+:::note
+Honeycomb ships with a modified version of the "simple reaction time task" from the [jsPsych tutorial](https://www.jspsych.org/7.3/tutorials/rt-task/). In the next section we'll create a new task and tell Honeycomb to run it!
+:::
 
 ## Edit the Task
 
-Now that the task is up and running we can make our first changes to the code! We'll edit the `package.json` file to reflect your information.
+Now that the project is up and running we can make our first changes to the code!
+
+### 1) Edit the Project Metadata
 
 1. Create a new branch

```

Comment:
Do you need to specify that this command is run in the Terminal and without killing the server?

Reply:
I mean, you could rerun it, but maybe folks need to be reminded that the server needs to be running constantly or else the app will close?

Reply:
I think it's okay to leave it as is. Either closing the server and changing branches or changing branches in a new terminal will work.  We document that the `npm run dev` will reflect the changes immediately so whether or not they keep it running is up to them

Reply:
I updated the text to say "Create a new branch in a separate terminal" which should help a little